src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs

```diff
@@ -146,13 +146,13 @@ public async Task BeforeStartAsync(DistributedApplicationModel appModel, Cancell
 
                         var http = daprCli.GetEndpoint("http");
                         var grpc = daprCli.GetEndpoint("grpc");
-
+                        
                         context.EnvironmentVariables.TryAdd("DAPR_GRPC_ENDPOINT", grpc);
                         context.EnvironmentVariables.TryAdd("DAPR_HTTP_ENDPOINT", http);
                     }));
 
-            daprCli.Annotations.Add(new EndpointAnnotation(ProtocolType.Tcp, name: "grpc", port: sidecarOptions?.DaprGrpcPort));
-            daprCli.Annotations.Add(new EndpointAnnotation(ProtocolType.Tcp, name: "http", port: sidecarOptions?.DaprHttpPort));
+            daprCli.Annotations.Add(new EndpointAnnotation(ProtocolType.Tcp, name: "grpc", port: sidecarOptions?.DaprGrpcPort, uriScheme: "http"));
+            daprCli.Annotations.Add(new EndpointAnnotation(ProtocolType.Tcp, name: "http", port: sidecarOptions?.DaprHttpPort, uriScheme: "http"));
             daprCli.Annotations.Add(new EndpointAnnotation(ProtocolType.Tcp, name: "metrics", port: sidecarOptions?.MetricsPort));
```

Comment:
@philliphoff do you know what the uri schema for metrics and profiling should be?

Reply:
Both endpoints are HTTP-based, with the metrics endpoint being a basic HTTP endpoint and, I believe, the profiling endpoint being gRPC.

Reply:
found that in the docs for the profiling option:

| Name | Environment Variable |  Default | Description |
| ------ | ------------------------  | ------- | ------------- |
| --enable-profiling |   | false | Enable “pprof” profiling via an HTTP endpoint |


So it's HTTP too. Changed it accordingly.